### ISSUE TITLE
LearnDash course pages not displaying properly, had to disable Elementor to fix it

### ISSUE THREAD

**Issue by deborahfelicio** [elementor/elementor#13725]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a default WordPress theme active.
- [ ] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

I am creating a new wordpress installation with LearnDash. After installing Elementor, LearnDash elements don't display at all.
![learnDash](https://user-images.githubusercontent.com/74010655/106153691-b8d60100-6176-11eb-9b53-d56872e7a5f4.png)
![workingLearnDash](https://user-images.githubusercontent.com/74010655/106153693-b96e9780-6176-11eb-86bc-de98932d90d3.png)

**Comment by udidol** [comment]
Hi @deborahfelicio,
Could you provide a more detailed explanation of your problem?
Could you provide detailed steps to reproduce your issue? What parts of LearnDash don't work?

**Comment by deborahfelicio** [comment]
The LearnDash features are not compromised, it is only the displaying of the course pages that looses the extra LearnDash elements. The pictures comparing the way the page looks when elementor is active (all blank), the other one with Elementor inactive, all the LearnDash elements being displayed.

**Comment by DorShahar** [comment]
Hi @deborahfelicio ,
Thanks for reaching out. We are aware ti the problem and working to fix it. 
I'm merging this issue with #13706 for better tracking.

Thanks

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.1.1

Feel free to update

Thanks!
